### PR TITLE
(Ready) Translate Card/70005.txt (Cardmaster Taiga)

### PR DESCRIPTION
**Pull request by Noitcereon**
Translation of the boy cardmaster, Taiga.

Translation done with <https://www.deepl.com/en/translator> and manual adjustments.

### CODE REVIEW

File: CSV/Language/English/Card/70005.txt [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
-少しは強くなったか？まあ今回もボクの勝ちだろうけどな！,
-兄ちゃん強いけどな、今度はボクが絶対に勝つ！さあバトルしようぜ！,
-強えぇ…。くっ、くそぉ…。次は負けないぜ！,
-よっしゃ、ボクの勝ちだ！兄ちゃん、バトルありがとな！,
-兄ちゃんのターンからだな！さあバトルスタートだ！,
-さあ、バトル開始だ！,
-ドローするぜ。,
-ドロー！,
+タイガ,Taiga
+わわ、兄ちゃん、もしかして無敗のルーキーか？\nすげぇ、こんなところで会えるなんてびっくりだぜ。,Whoa; aren't you the undefeated rookie? That's awesome; I'm surprised to see you here.
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Card/70005.txt:3]: 'That's awesome! I'm surprised to see you here.'

File: CSV/Language/English/Card/70005.txt [hunk 2]
```diff
@@ -1,43 +1,43 @@
...
+ボクはタイガ。ギルドに入って3ヵ月のF級カードマスターだ！,My name is Taiga. I'm an F-rank cardmaster; and I've been in the guild for three months!
+ボクの姉ちゃんはC級のカードマスターなんだぜ。\n姉ちゃんに追いつくためにもこの大会で絶対に勝ちまくる！,My sister is a C-rank cardmaster. I'm going to win this tournament to catch up with her!
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Card/70005.txt:5]: Maybe 'win this tournament and catch up with her' might sound nicer.

File: CSV/Language/English/Card/70005.txt [hunk 3]
```diff
@@ -1,43 +1,43 @@
...
+よお、兄ちゃん。その指輪をしているってことはカードマスターだな！,Hey. You're wearing one of those rings; which means you're a cardmaster!
+ボクの姉ちゃんはC級のカードマスターなんだぜ。姉ちゃんに追いつくためにもこの大会で絶対に勝ちまくる！,My sister is a C-rank cardmaster. I'm going to win this tournament to catch up with her!
+お、兄ちゃん。また会ったな。,Oh. We meet again.
+兄ちゃんすげえよな。ボクの姉ちゃんにも勝っちまうんだもん。\nボク、兄ちゃんにあこがれて、最近修行頑張ってんだ！,You're awesome. You even beat my sister. I've been training hard lately because I want to be like you.
+だから、今日は修行の成果を見てくれよ！\n獣デッキで兄ちゃん相手にもいい勝負してみせるぜ！,So today; you'll see the results of my training! I'll give you a good fight with my beast deck!
+少しは強くなったか？まあ今回もボクの勝ちだろうけどな！,You're a bit stronger now? Well; I'm sure I'll win again this time!
+兄ちゃん強いけどな、今度はボクが絶対に勝つ！さあバトルしようぜ！,You're strong; but this time I'll definitely win! Come on; let's fight!
+強えぇ…。くっ、くそぉ…。次は負けないぜ！,So strong... Damn... I won't lose next time!
+よっしゃ、ボクの勝ちだ！兄ちゃん、バトルありがとな！,Yes; I've won! Thanks for the fight!
+兄ちゃんのターンからだな！さあバトルスタートだ！,It's your turn to start! Come on; let's begin!
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Card/70005.txt:16]: Maybe 'let's go' to match her personality a little more.

File: CSV/Language/English/Card/70005.txt [hunk 4]
```diff
@@ -1,43 +1,43 @@
...
-さあ、バトル開始だ！,
-ドローするぜ。,
-ドロー！,
+タイガ,Taiga
+わわ、兄ちゃん、もしかして無敗のルーキーか？\nすげぇ、こんなところで会えるなんてびっくりだぜ。,Whoa; aren't you the undefeated rookie? That's awesome; I'm surprised to see you here.
+ボクはタイガ。ギルドに入って3ヵ月のF級カードマスターだ！,My name is Taiga. I'm an F-rank cardmaster; and I've been in the guild for three months!
+ボクの姉ちゃんはC級のカードマスターなんだぜ。\n姉ちゃんに追いつくためにもこの大会で絶対に勝ちまくる！,My sister is a C-rank cardmaster. I'm going to win this tournament to catch up with her!
+てなわけで、勝負しようぜ！,So let's get on with the fight!
+よお、兄ちゃん。その指輪をしているってことはカードマスターだな！,Hey. You're wearing one of those rings; which means you're a cardmaster!
+ボクの姉ちゃんはC級のカードマスターなんだぜ。姉ちゃんに追いつくためにもこの大会で絶対に勝ちまくる！,My sister is a C-rank cardmaster. I'm going to win this tournament to catch up with her!
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Card/70005.txt:8]: Same as above.

File: CSV/Language/English/Card/70005.txt [hunk 5]
```diff
@@ -1,43 +1,43 @@
...
-永続魔法、{使用カード}を発動！獣モンスターたちの攻撃力を永続的にパワーアップさせる！,
+へへん、{使用カード}を上級召喚だ！,Heheh; {使用カード} is an advanced summon!
+{使用カード}を召喚だ。,Summon {使用カード}.
+さらにもう1枚発動だ！,And another one is activated!
+{使用カード}の効果発動！手札からレベル３以下のモンスター１体を特殊召喚する！,Activating the effect of {使用カード}! I can now Special Summon a Level 3 or lower monster from my hand!
+魔法カード発動！{使用カード}！デッキから地属性のカードを加えるぜ.,Magic card activated! {使用カード}! Adding a monster card to my hand from my deck.
+手札にモンスターを呼ぶぜ。{使用カード}発動！,I'm gonna add a monster card to my hand. {使用カード} activation!
+魔法カード発動！{使用カード}！HPを強化だ！,Magic card activated! I'm using {使用カード} to increase the HP of my target! (translate check needed)
+体力を上げて頑丈にするぜ。{使用カード}を発動！,I'm gonna boost the sturdyness of my target. Activate {使用カード}! (translate check needed)
+切り札のカードだ！{使用カード}を発動！獣モンスターを永続的にパワーアップ！,This is my trump card! Activate {使用カード}! It will permanently power up beast monsters!
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Card/70005.txt:29]: 'my beast monsters.'

File: CSV/Language/English/Card/70005.txt [hunk 6]
```diff
@@ -1,43 +1,43 @@
...
+永続魔法、{使用カード}を発動！獣モンスターたちの攻撃力を永続的にパワーアップさせる！,Activate permanent magic; {使用カード}! This will power up the attack power of my beast monsters while it remains on the field!
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Card/70005.txt:30]: Maybe 'permanent spell'

File: CSV/Language/English/Card/70005.txt [hunk 7]
```diff
@@ -1,43 +1,43 @@
...
 装備魔法カード発動！{使用カード}！,Magic card activate! {使用カード}!
-魔法カードを発動するぜ！{使用カード}！,
-トラップカードを発動！{使用カード}！,
-カードをセットするぜ。,
-カードを伏せるぜ。,
-ボクの攻撃！{防御カード}を攻撃！,
+魔法カードを発動するぜ！{使用カード}！,I'm gonna activate a magic card! {使用カード}!
+トラップカードを発動！{使用カード}！,Activate trap card! {使用カード}!
+カードをセットするぜ。,Let's set the cards.
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Card/70005.txt:34]: 'Let's set some cards' or 'I'll set these cards.'

File: CSV/Language/English/Card/70005.txt [hunk 8]
```diff
@@ -1,43 +1,43 @@
...
+カードを伏せるぜ。,I'm gonna put down my cards.
```
Comment (HotHeadEmperor) [inline CSV/Language/English/Card/70005.txt:35]: 'place' instead of 'put down'

File: CSV/Language/English/Card/70005.txt [hunk 9]
```diff
@@ -1,43 +1,43 @@
...
-よっしゃ、ボクの勝ちだ！兄ちゃん、バトルありがとな！,
-兄ちゃんのターンからだな！さあバトルスタートだ！,
-さあ、バトル開始だ！,
-ドローするぜ。,
-ドロー！,
+タイガ,Taiga
+わわ、兄ちゃん、もしかして無敗のルーキーか？\nすげぇ、こんなところで会えるなんてびっくりだぜ。,Whoa; aren't you the undefeated rookie? That's awesome! I'm surprised to see you here.
+ボクはタイガ。ギルドに入って3ヵ月のF級カードマスターだ！,My name is Taiga. I'm an F-rank cardmaster; and I've been in the guild for three months!
+ボクの姉ちゃんはC級のカードマスターなんだぜ。\n姉ちゃんに追いつくためにもこの大会で絶対に勝ちまくる！,My sister is a C-rank cardmaster. I'm going to win this tournament and catch up with her!
+てなわけで、勝負しようぜ！,So let's get on with the fight!
```
Comment (Giratena) [inline CSV/Language/English/Card/70005.txt:6]: This sentence feels a little flat and doesn't quite flow with the previous sentence.

Maybe something like, "So get ready to battle! I'm going to go all out!"?

Comment (Noitcereon) [reply CSV/Language/English/Card/70005.txt:6]: What about `"So get ready to battle! I'm gonna go all out!"`?

Comment (Giratena) [reply CSV/Language/English/Card/70005.txt]: Perfect! Couldn't have worded it better myself ^^

**Comment by Giratena** [comment]
Merging this in now, thank you for the help @Noitcereon and @HotHeadEmperor!

**Pull request merged**